services/worker-manager/src/estimator.js

```diff
@@ -21,15 +21,26 @@ class Estimator {
     // how many extra we want if we do want any
     const requestedCapacity = Math.max(0, desiredCapacity - runningCapacity);
 
-    this.monitor.log.simpleEstimate({
+    const estimatorInfo = {
       workerPoolId,
       pendingTasks,
       minCapacity,
       maxCapacity,
       runningCapacity,
       desiredCapacity,
       requestedCapacity,
-    });
+    }
+
+    let overProvisioned = false;
+    if (runningCapacity > (maxCapacity * 1.25)) {
```

Comment:
I understand this is arbitrary, but maybe we should add a comment on why and how this number